test-cases/performance/perf-scylla-cassandra-uniform.yaml

```diff
@@ -0,0 +1,5 @@
+stress_cmd_m: ["cassandra-stress mixed no-warmup cl=QUORUM duration=60m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=100 throttle=12500/s' -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=UNIFORM(1..1000000000)' ",
+               "cassandra-stress mixed no-warmup cl=QUORUM duration=60m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=150 throttle=25000/s' -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=UNIFORM(1..1000000000)' ",
+               "cassandra-stress mixed no-warmup cl=QUORUM duration=60m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate 'threads=200 throttle=40000/s' -col 'size=FIXED(128) n=FIXED(8)' -pop 'dist=UNIFORM(1..1000000000)' "]
+
+compaction_strategy: "LeveledCompactionStrategy"
```

Comment:
if this file is never called by itself (always together with `perf-scylla-cassandra-regular.yaml` and this parameter is already defined in the other file, this can be removed

Reply:
better to same, to be sure that with this test LCS will be used